### PR TITLE
[Ready for review] Polish translation  for Accessibility Fundamentals Overview

### PR DESCRIPTION
**Pull request by zwiastunsw**
I added polish translation
fix https://github.com/w3c/wai-translations/issues/198

### CODE REVIEW

File: content/index.pl.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+inline_css: |
+  img.icon {--img-width:1em}
+  
+---
+
+Poniższe materiały zawierają informacje ułatwiające **rozpoczęcie pracy z dostępnością**:
+
+[[Wprowadzenie do dostępności internetowej]](/fundamentals/accessibility-intro/)
+:   Przedstawia dostępność internetową i łącza do dodatkowych materiałów. Obejmuje dostępność w kontekście; dlaczego dostępność jest ważna dla osób indywidualnych, firm i społeczeństwa; udostępnianie technologii cyfrowej; ocenę dostępności; i  proste przykłady.
```
Comment (viking-pl) [inline content/index.pl.md:41]:
```suggestion
:   Przedstawia dostępność internetową oraz łącza do dodatkowych materiałów. Obejmuje dostępność w kontekście jej znaczenia dla zwykłych ludzi, firm i społeczeństwa; dostępu do technologii cyfrowej, oceny dostępności oraz proste przykłady.
```

File: content/index.pl.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    -   [Film Wprowadzenie do dostępności internetowej i standardów W3C](/videos/standards-and-benefits/) 4 minuty {% include image.html src="video.svg" alt="" class="icon" %}
+
+[[Dostępność: Chodzi o ludzi]](/people/)
```
Comment (viking-pl) [inline content/index.pl.md:44]:
```suggestion
[[Dostępność jest dla ludzi]](/people/)
```

File: content/index.pl.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+:   Przedstawia ludzki wymiar dostępności i rolę dostępności w różnorodności, równości i włączeniu. Łącza do:
+    -   [Perspektywy dostępności internetowej: Poznaj wpływ i korzyści dla wszystkich](/perspective-videos/) - filmy pokazujące, że dostępność internetowa jest niezbędna dla osób z niepełnosprawnościami i użyteczna dla wszystkich. Dziesięć 1-minutowych filmów i 7-minutowa kompilacja {% include image.html src="video.svg" alt="" class="icon" %}
```
Comment (viking-pl) [inline content/index.pl.md:46]:
```suggestion
    -   [Walory dostępności internetowej: Poznaj wpływ i korzyści dla wszystkich](/perspective-videos/) - filmy pokazujące niezbędność dostępności internetowej dla osób niepełnosprawnych  oraz jej użyteczność dla wszystkich. Dziesięć jednominutowych filmów i siedmiominutowa kompilacja {% include image.html src="video.svg" alt="" class="icon" %}
```

File: content/index.pl.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    -   [[Jak osoby z niepełnosprawnością korzystają z Internetu]](/people-use-web/)<!-- , [Filmy pokazujące, jak osoby z niepełnosprawnością korzystają z Internetu](/people-use-web/) Osiemnaście ponad 2-minutowych filmów i 3 kompilacje {% include image.html src="video.svg" alt="" class="icon" %} -->
```
Comment (viking-pl) [inline content/index.pl.md:47]:
```suggestion
   -   [[Jak osoby z niepełnosprawnością korzystają z Internetu]](/people-use-web/)<!-- , [Filmy demonstrujące korzystanie z Internetu przez osoby niepoełnosprawne](/people-use-web/) Osiemnaście ponad dwuminutowych filmów i 3 kompilacje {% include image.html src="video.svg" alt="" class="icon" %} -->
```

File: content/index.pl.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    -   [Starsi użytkownicy i dostępność Internetu](/older-users/) 
+    -   [[Dostępność, użyteczność i integracja]](/fundamentals/accessibility-usability-inclusion/)
+
+[Podstawowe komponenty dostępności internetowej](/fundamentals/components/)
+:   Przedstawia, w jaki sposób dostępność internetowa zależy kilku współpracujących ze sobą komponentów tworzenia treści i interakcji oraz pokazuje związek między wytycznymi WAI a innymi standardami.
```
Comment (viking-pl) [inline content/index.pl.md:52]:
```suggestion
:   Przedstawia zależność dostępności internetowej od kilku współpracujących ze sobą komponentów tworzenia treści i interakcji oraz pokazuje związek między wytycznymi WAI a innymi standardami.
```

File: content/index.pl.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+[[Zasady dostępności]](/fundamentals/accessibility-principles/)
+:   Zwięzłe wprowadzenie do wymagań dostępności dla stron internetowych, aplikacji, przeglądarek i innych narzędzi. Jest zorganizowana podobnie, jak standardów dostępności WAI, i stanowi użyteczny pomost do standardów. Przeczytaj tę stronę przed zapoznaniem się ze standardami/wytycznymi.
```
Comment (viking-pl) [inline content/index.pl.md:55]:
```suggestion
:   Zwięzłe wprowadzenie do wymagań dostępności dla stron internetowych, aplikacji, przeglądarek i innych narzędzi. Ma podobną strukturę do standardów dostępności WAI i stanowi użyteczne do nich wprowadzenie. Przeczytaj tę stronę przed zapoznaniem się ze standardami/wytycznymi.
```

File: content/index.pl.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+Badania:
+:   -   [[Wymagania użytkowników dotyczące dostępności]](/research/user-requirements/) - Dokumentacja badań nad potrzebami użytkowników w środowiskach wirtualnych lub immersyjnych (XR), komunikacji w czasie rzeczywistym (RTC), interfejsach języka naturalnego, multimediach, słabym wzroku, dostępności poznawczej i innych.
```
Comment (viking-pl) [inline content/index.pl.md:58]:
```suggestion
:   -   [[Cyfrowa dostępność dla użytkowników]](/research/user-requirements/) - Dokumentacja badań nad potrzebami użytkowników dotyczących środowisk wirtualnych lub immersyjnych (XR), komunikacji w czasie rzeczywistym (RTC), interakcji w języku naturalnym, multimediów, użytkowników: o słabym wzroku, dostępności poznawczej i innych.
```

File: content/index.pl.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+    -   [[Sympozjum dotyczące badań nad sztuczną inteligencją (AI) i dostępnością 2023]](/research/ai2023/) - Streszczenia, transkrypcje i filmy z międzynarodowego sympozjum poświęconego pozytywnym i negatywnym skutkom sztucznej inteligencji (AI) na dostępność cyfrową.
```
Comment (viking-pl) [inline content/index.pl.md:59]:
```suggestion
    -   [[Sympozjum dotyczące badań nad dostępnością i sztuczną inteligencją (AI)  2023]](/research/ai2023/) - Streszczenia, transkrypcje i filmy z międzynarodowego sympozjum poświęconego pozytywnym i negatywnym skutkom sztucznej inteligencji (AI) na dostępność cyfrową.
```

File: content/index.pl.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+Kursy dostępności cyfrowej:
+:   -   [[Podstawy dostępności cyfrowej &mdash;  bezpłatny kurs internetowy]](/fundamentals/foundations-course/) - Przedstawia kurs we własnym tempie opracowany przez W3C WAI dla programistów, projektantów, UX, autorów, menedżerów, rzeczników. **Zaprojektowany dla osób z wykształceniem technicznym i nietechnicznym**, w tym studentów, instruktorów, specjalistów i osób z niepełnosprawnością. Kurs jest bezpłatny z opcjonalnym certyfikatem.
```
Comment (viking-pl) [inline content/index.pl.md:62]:
```suggestion
:   -   [[Podstawy dostępności cyfrowej &mdash;  bezpłatny kurs internetowy]](/fundamentals/foundations-course/) - Przedstawia kurs opracowany przez W3C WAI dla programistów, projektantów, UX, autorów, menedżerów, rzeczników, do zrealizowania we własnym tempie. **Zaprojektowany dla osób z umysłem technicznym i nietechnicznym**, w tym studentów, instruktorów, specjalistów i osób z niepełnosprawnością. Kurs jest bezpłatny z opcjonalnym certyfikatem.
```

File: content/index.pl.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+    -   [[Lista kursów z zakresu edukacji, szkoleń i certyfikacji w zakresie dostępności cyfrowej]](/courses/list/) - zawiera listę kursów od różnych dostawców.
```
Comment (viking-pl) [inline content/index.pl.md:63]:
```suggestion
    -   [[Lista kursów o dostępności cyfrowej z zakresu edukacji, szkoleń i certyfikacji]](/courses/list/) - zawiera listę kursów od różnych dostawców.    
```

**Review by viking-pl — CHANGES_REQUESTED** [review]
Those 10 tweaks are all my suggestions. Nothing major, just easier, more natural to read.

File: content/index.pl.md (hunk 2 above)
Comment (zwiastunsw) [reply content/index.pl.md:44]:
```suggestion
[[Dostępność: Jest dla ludzi]](/people/)
```
I think the structure of the title needs to be preserved in this case as well. The use of a colon here signals the importance of the following statement

File: content/index.pl.md (hunk 3 above)
Comment (zwiastunsw) [reply content/index.pl.md:46]:
```suggestion
    -   [Walory dostępności internetowej: Poznaj wpływ i korzyści dla wszystkich](/perspective-videos/) - filmy pokazujące niezbędność dostępności internetowej dla osób z niepełnosprawnościami oraz jej użyteczność dla wszystkich. Dziesięć jednominutowych filmów i siedmiominutowa kompilacja {% include image.html src="video.svg" alt="" class="icon" %}
```
Minor correction: "dla osób z niepełnosprawnościami" not "osób niepełnosprawnych" (inclusive language)

File: content/index.pl.md (hunk 2 above)
Comment (viking-pl) [reply content/index.pl.md:44]: "dostępność" does't work with "o" IMHO.
```suggestion
[[Dostępność jest dla ludzi]](/people/)
```
I don't think we need to copy the English interpunction. It does not bring the same semantics or ornament when written in polish language.

**Comment by remibetin** [comment]
Hi @zwiastunsw 

The preview is broken. This is due to the indentation line 13: removing the leading space before `contributors` should fix the problem.

FYI, if you "[Allow edits from maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)", I will be able to directly fix these type of problems when I see them.

**Review by viking-pl — APPROVED** [review]

**Review by remibetin — APPROVED** [review]
Thanks @zwiastunsw!

**Comment by zwiastunsw** [comment]
Hi @remibetin: Hold the publications for a while yet. We are considering how best to translate the title : Web Accessibility Perspectives ("Walory dostępności internetowej" or "Perspektywy dostępności internetowej" or "Potencjał dostępności internetowej". This is an important title and we want to give it the best possible shape.

**Comment by remibetin** [comment]
@zwiastunsw Oops, already published 🙈 

When you have found the best possible translation for "Web Accessibility Perspectives", feel free to open a new PR and I will publish the update as quickly as possible!

Thanks again!

**Comment by remibetin** [comment]
@zwiastunsw You may also want to update the "Video Introduction" translation that links to the "Web Accessibility Perspective Videos" resource.

**Pull request merged**